### PR TITLE
NMS-16033: Update docs on default RRD

### PR DESCRIPTION
**Pull request by mmahacek**
Consolidate RRD text across poller and collect configs by reusing the same text on both.

### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16033

### CODE REVIEW

**Review by dino2gnt — APPROVED** [review]
LGTM

File: docs/modules/operation/pages/deep-dive/retention.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   <rra>RRA:AVERAGE:0.5:12:1488</rra><3>
+   <rra>RRA:AVERAGE:0.5:288:366</rra><4>
+   <rra>RRA:MAX:0.5:288:366</rra><5>
+   <rra>RRA:MIN:0.5:288:366</rra><6>
+</rrd>
+----
+<1> Step of 300 seconds (5 minutes) per bucket.
+This value should match the interval of the services in the package.
+<2> Store the average of 1 bucket for 2016 buckets.
+5 minute averages kept for 7 days.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/retention.adoc:20]:
```suggestion
5-minute averages kept for 7 days.
```

File: docs/modules/operation/pages/deep-dive/retention.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<3> Store the average of 12 buckets for 1488 buckets.
+1 hour average kept for 62 days.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/retention.adoc:22]:
```suggestion
1-hour average kept for 62 days.
```

File: docs/modules/operation/pages/deep-dive/retention.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<4> Store the average of 288 buckets for 366 buckets.
+1 day average kept for 366 days.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/retention.adoc:24]:
```suggestion
1-day average kept for 366 days.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
a few minor changes.

**Review by Bonrob2 — DISMISSED** [review]
LGTM! Thanks!

File: docs/modules/operation/pages/deep-dive/retention.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
+When using RRD/JRB storage, data is stored in the file system using a RRD-type strategy.
+With this "round robin" storage, data is kept in aggregated time buckets, with the latest bucket overwriting the oldest bucket, and multiple collections within the time period for a bucket will cause the data to be consolidated based on the type of bucket.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/retention.adoc:2]:
```suggestion
With this "round robin" storage, data is kept in aggregated time buckets, with the latest bucket overwriting the oldest bucket.
Multiple collections within the time period for a bucket will cause the data to be consolidated based on the type of bucket.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**